### ISSUE TITLE
Add a "force move" function for soldiers

### ISSUE THREAD

**Issue by krichter722** [jsettlers/settlers-remake#506]
Soldiers are attracted by attack targets quite intensively which is useful for unsupervised attacks, but prevents them from (efficient) retreat. Keeping the (sometimes) useful attraction and giving the possibility for retreat can be achieved with a force function for moving soldiers.

experienced with 0.3-alpha

**Comment by DerKrawallkeks** [comment]
I agree, but didn't I already open dich an issue¿ Not sure;) anyway, I agree and it's very annoying to play

**Comment by DerKrawallkeks** [comment]
I agree, but didn't I already open dich an issue¿ Not sure;) anyway, I agree and it's very annoying to retreat. Would be a great improvement

**Comment by CookingCookie** [comment]
Will this commit be merged into master someday or is it not working?
